src/main/java/blackjack/view/OutputView.java

```diff
@@ -0,0 +1,82 @@
+package blackjack.view;
+
+import blackjack.domain.GameResult;
+import blackjack.domain.ResultType;
+import blackjack.domain.card.Card;
+import blackjack.domain.participant.BlackJackParticipant;
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Player;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OutputView {
+
+    private static final String NAME_DELIMITER = ", ";
+
+    public static void printInputNames() {
+        System.out.println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)");
+    }
+
+    public static void printGameInitializeMessage(List<BlackJackParticipant> participants, int startingCardCount) {
+        String participantNames = participants.stream()
+                .map(BlackJackParticipant::getName)
+                .collect(Collectors.joining(NAME_DELIMITER));
+        System.out.println("\n" + participantNames + "에게 " + startingCardCount + "장의 카드를 나누었습니다.");
```

Comment:
OS마다 개행 문자가 다르답니다. 그래서 자바에서는 System.lineSeparator()를 제공해주죠.
```java
    /**
     * Returns the system-dependent line separator string.  It always
     * returns the same value - the initial value of the {@linkplain
     * #getProperty(String) system property} {@code line.separator}.
     *
     * <p>On UNIX systems, it returns {@code "\n"}; on Microsoft
     * Windows systems it returns {@code "\r\n"}.
     *
     * @return the system-dependent line separator string
     * @since 1.7
     */
```
자바독 내용입니다. 😉